charts/kuberpult/values.yaml.tpl

```diff
@@ -17,13 +17,13 @@ log:
 cd:
   image: kuberpult-cd-service
   backendConfig:
-    create: false # Add backend config for health checks on GKE only
+    create: false  # Add backend config for health checks on GKE only
 frontend:
   image: kuberpult-frontend-service
 ingress:
   annotations: {}
-  domainName: null
-  exposeReleaseEndpoint: false
+  domainName: kuberpult.example.com
+  exposeReleaseEndpoint: true
```

Comment:
I don't think we need to change the release endpoint.

Reply:
Yes that's true, I just tried it, I will move it back to default